packages/rich-text/src/plugins/Marks/helpers.ts

```diff
@@ -1,9 +1,22 @@
 import { MARKS } from '@contentful/rich-text-types';
 import { HotkeyPlugin, isMarkActive, KeyboardHandler, toggleMark } from '@udecode/plate-core';
+import type { PlateEditor } from '@udecode/plate-core';
 import isHotkey from 'is-hotkey';
 
 import { RichTextEditor } from '../../types';
 
+export const toggleMarkAndDeactivateConflictingMarks = (
+  editor: PlateEditor<RichTextEditor>,
+  mark: MARKS
+) => {
+  toggleMark(editor, { key: mark });
+  if (mark === MARKS.SUBSCRIPT && isMarkActive(editor, MARKS.SUPERSCRIPT)) {
+    toggleMark(editor, { key: MARKS.SUPERSCRIPT });
+  } else if (mark === MARKS.SUPERSCRIPT && isMarkActive(editor, MARKS.SUBSCRIPT)) {
+    toggleMark(editor, { key: MARKS.SUBSCRIPT });
+  }
```

Comment:
How about using the `clear` option of `toggleMark`?

Reply:
I'm not sure how the clear option works, but would that remove any other marks there (such as italic)?

Reply:
So we can have something like below:

```javascript
const subs = [MARKS.SUPERSCRIPT, MARKS.SUBSCRIPT]

const clear = subs.includes(mark) ? subs | []

toggleMark(editor, {key: mark, clear})
```

Reply:
The docs don't exactly make it clear how to use this it accepts a string or a list of strings. I've tried 
`toggleMark(editor, { key: MARKS.SUBSCRIPT, clear: MARKS.SUPERSCRIPT });` without any success.

Do you have any suggestions?

Reply:
Ah sorry the comment didn't update until after I wrote mine

Reply:
This indeed did work!